tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -1378,9 +1380,11 @@ static void handleRequestIndex(const RequestDict &Req,
     return;
 
   handleSemanticRequest(Req, Rec, [Req, Rec]() {
-    auto SourceFile = getPrimaryFileForRequestOrEmitError(Req, Rec);
+    Optional<StringRef> SourceFile =
+        getSourceFileForRequestOrEmitError(Req, Rec);
```

Comment:
Since this is now returning the file name what do you think about renaming the function to `getSourceFileNameForRequestOrEmitError`?